from django.core.exceptions import ValidationError\n\ndef validate_file_type(file):\n    valid_mime_types = ['text/csv', 'application/vnd.ms-excel']\n    file_mime_type = file.content_type\n\n    if file_mime_type not in valid_mime_types:\n        raise ValidationError('Unsupported file type.')\n\ndef handle_uploaded_file(f):\n    validate_file_type(f)\n    # Save the file and process it\n    with open('some/file/path.csv', 'wb+') as destination:\n        for chunk in f.chunks():\n            destination.write(chunk)\n    process_csv('some/file/path.csv')